nautobot/core/testing/filters.py

```diff
@@ -88,11 +88,26 @@ def get_q_filter(self):
             return self.filterset.declared_filters["q"].filter_predicates
 
         def test_id(self):
-            """Verify that the filterset supports filtering by id."""
-            params = {"id": self.queryset.values_list("pk", flat=True)[:2]}
-            filterset = self.filterset(params, self.queryset)
-            self.assertTrue(filterset.is_valid())
-            self.assertEqual(filterset.qs.count(), 2)
+            """Verify that the filterset supports filtering by id with only lookup `__n`."""
+            with self.subTest("Assert `id`"):
+                params = {"id": self.queryset.values_list("pk", flat=True)[:2]}
```

Comment:
Perhaps to fix the MySQL failures:

```suggestion
                params = {"id": list(self.queryset.values_list("pk", flat=True)[:2])}
```

and similar for the other subtests below?